server/app/services/geo/ServiceAreaInclusionGroup.java

```diff
@@ -0,0 +1,40 @@
+package services.geo;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Contains methods for serializing and deserializing stored values for {@link
+ * ServiceAreaInclusion}.
+ */
+public final class ServiceAreaInclusionGroup {
+  private ServiceAreaInclusionGroup() {}
+
+  public static ImmutableList<ServiceAreaInclusion> deserialize(String serviceAreas) {
```

Comment:
**suggestion (blocking):** document the serialization format

Reply:
Added javadoc for deserialize.